src/librustc_infer/infer/canonical/query_response.rs

```diff
@@ -671,6 +671,10 @@ impl<'tcx> TypeRelatingDelegate<'tcx> for QueryTypeRelatingDelegate<'_, 'tcx> {
         });
     }
 
+    fn const_equate(&mut self, _a: &'tcx Const<'tcx>, _b: &'tcx Const<'tcx>) {
+        unimplemented!()
```

Comment:
Is this intended for a follow up? Do we know what's blocking this?

Reply:
This was is from @skinny121, will have to look into this.

Reply:
This seems like it is unreachable rn. `QueryTypeRelatingDelegate` is only used with `TypeRelating`. `TypeRelating`  should never use `const_equate` afaict.

Replaced it with a `span_bug` :thinking: We can probably ignore this for now